File: Assignment_14/assignment4.c
/////////////////////////////////////////////////////////
//
//  File name :     assignment4.c
//  Description :   Accept number of rows and columns from user and display below pattern.
//  Author :        Aryan S. Pardeshi.
//  Date :          04/06/2025
//
/////////////////////////////////////////////////////////

// Input  :  iRow = 5   iCol = 5
// Output :  1  2  3  4  5
//          -1 -2 -3 -4 -5
//           1  2  3  4  5
//          -1 -2 -3 -4 -5
//           1  2  3  4  5

#include <stdio.h>

void Pattern(int iRow, int iCol) {
    for (int i = 1; i <= iRow; i++) {
        for (int j = 1; j <= iCol; j++) {
            if (i % 2 == 0)
                printf("-%d\t", j);
            else
                printf("%d\t", j);
        }
        printf("\n");
    }
}

int main() {
    int iValue1 = 0, iValue2 = 0;

    printf("Enter the number of Rows and Columns: ");
    scanf("%d %d", &iValue1, &iValue2);

    Pattern(iValue1, iValue2);

    return 0;
}

File: Assignment_47/assignment4.java
//////////////////////////////////////////////////////////////////////////
// 
//    File name    :- assignment4.java
//    Description  :- Write a program which accepts number of rows and 
//                    columns from user and display below pattern.
//    Author name  :- Aryan Shailendrasingh Pardeshi
//    Date         :- 19/08/2025
//
//////////////////////////////////////////////////////////////////////////

// INPUT:  iRow = 3, iCol = 4

/* OUTPUT:
            *   #   *   #   
            *   #   *   #   
            *   #   *   #
*/

import java.util.*;

class Pattern
{
    public void Pattern(int iRow, int iCol)
    {
        for(int i = 1; i <= iRow; i++)  
        {
            for(int j = 1; j <= iCol; j++)  
            {
                if(j % 2 == 1) 
                {
                    System.out.print("*\t");
                }
                else           
                {
                    System.out.print("#\t");
                }
            }
            System.out.println();
        }
    }
}

class assignment4
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int iValue1 = 0, iValue2 = 0;

        System.out.print("Enter number of rows : ");
        iValue1 = sobj.nextInt();

        System.out.print("Enter number of columns : ");
        iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();
        pobj.Pattern(iValue1, iValue2);
    }
}
